testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -284,21 +280,19 @@ def test_rdkit_kwargs(self):
 
 
     def test_coordinates(self):
+        # manually create the molecule to compare
```

Comment:
I think it's worth to expand this comment to say why this is necessary, as explained in https://github.com/MDAnalysis/mdanalysis/pull/4640#issuecomment-2242695321.

Reply:
Something like this?

Also this is a merge approval @RMeli ? you didn't green tick so it won't auto merge 😅

Reply:
Perfec, thanks!